VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationEditTextView.java

```diff
@@ -99,18 +101,40 @@ public boolean isValidInput() {
 
         // check if this is required field and has an input value
         if (mField.isRequired() && !hasValue()) {
-            handleError(mField.getErrorMessage().getRequired());
+            String errorMessage = mField.getErrorMessage().getRequired();
+            if(errorMessage==null || errorMessage.isEmpty()){
+                errorMessage = getView().getResources().getString(R.string.error_enter_field,
+                        mField.getLabel());
+            }
+            if(errorMessage!=null){
+                handleError(errorMessage);
+            }
             return false;
         }
 
         // check if length restrictions are followed
         int inputLength = getCurrentValue().getAsString().length();
         if (inputLength < mField.getRestriction().getMinLength()) {
-            handleError(mField.getErrorMessage().getMinLength());
+            String errorMessage = mField.getErrorMessage().getMinLength();
+            if(errorMessage==null || errorMessage.isEmpty()){
+                errorMessage = getView().getResources().getString(R.string.error_min_length,
+                        mField.getLabel(), mField.getRestriction().getMinLength());
+            }
+            if(errorMessage!=null){
+                handleError(errorMessage);
+            }
```

Comment:
Null check not required, <code>handleError(String)</code> checks for null and empty text.